### ISSUE TITLE
application/argo

### ISSUE THREAD

**Issue by wg102** [StatCan/argoflow-azure#15]
Compare our manifest and kubeflow's, with careful checking for any customization.
Update the argoflow-azure applications in distribution
Test
Duplicate the changes in the template.

**Comment by wg102** [comment]
Here is where i found the argo :
- Upstream in pipelines: manifests/kustomize/third-party/argo
- manifest:kfp-tekton/upstream/third-party/argo
- manifest/apps/pipeline/upstream/cluster-scoped-resources/third-party/argo/installs/namespace/cluster-scoped (and many similar)
- manifest/apps/pipeline/upstream/third-party/argo

- and upstream **pipelines/third_party/argo**

**Comment by wg102** [comment]
Things to add: 
kubeflow-manifest/kustomize/application/argo
There are 3 patches + kustomization file

**Comment by wg102** [comment]
This is getting a lot harder than planned. The code would not build. 
It was decided that since all argo files are now part of the pipelines, we would put the patches under /kubeflow/pipellines/argo.

Once that was done, each of the file needed to be tracked to where it is now.
cluster-role -> [workflow-controller-clusterrole.yaml](https://github.com/kubeflow/pipelines/blob/v1.7.0-alpha.3/manifests/kustomize/third-party/argo/upstream/manifests/cluster-install/workflow-controller-rbac/workflow-controller-clusterrole.yaml) -> also needed to go from `rbac.authorization.k8s.io/v1beta1` to `rbac.authorization.k8s.io/v1`
config-map.yaml ->  [workflow-controller-configmap-patch.yaml](https://github.com/kubeflow/pipelines/blob/v1.7.0-alpha.3/manifests/kustomize/third-party/argo/base/workflow-controller-configmap-patch.yaml)

**Comment by wg102** [comment]
After a chat with the team, we discovered that the argo patches may not be as useful anymore.
We came to this conclusion that the argo was deployed within pipelines, therefore the two issues are linked together https://github.com/StatCan/aaw-argoflow-azure/issues/12. The patches triggered issues that were not able to fix and after review, may not be needed. 

As of now, there will not be argo patches. This may change when we get to test the argocd on dev, if there are any issues.
